maintainers/maintainer-list.nix

```diff
@@ -10191,6 +10191,12 @@
     githubId = 56224949;
     name = "Mia Kanashi";
   };
+  kanielrkirby = {
+    email = "kanielrkirby@runbox.com";
+    github = "kanielrkirby";
+    githubId = 77940607;
+    name = "Kaniel Kirby";
+  };
```

Comment:
Could you split this change into a separate commit with message `maintainers: add kanielrkirby`?